src/cljs/sixsq/slipstream/webui/messages/views.cljs

```diff
@@ -40,66 +39,105 @@
+                        :success    true
+                        :style      top-right
+                        :on-dismiss #(dispatch [::message-events/hide])}
+            [ui/MessageHeader [ui/Icon {:name icon-name}] header "\u2001\u00a0"]
+            [:a {:on-click #(dispatch [::message-events/open-modal])} (@tr [:more-info])]]])))))
+
+
+(defn alert-modal
   []
-  (let [alert-message (subscribe [::message-subs/alert-message])]
+  (let [alert-message (subscribe [::message-subs/alert-message])
+        alert-display (subscribe [::message-subs/alert-display])]
     (fn []
       (if-let [{:keys [type header content]} @alert-message]
         (let [icon-name (type->icon-name type)
-              header-class (str "webui-" (name type))]
+              visible? (= :modal @alert-display)]
           [ui/Modal
            {:close-icon true
-            :open       (boolean @alert-message)
+            :open       visible?
             :on-close   #(dispatch [::message-events/hide])}
-           [ui/ModalHeader {:class-name header-class}
-            [ui/Icon {:size "big"
-                      :name icon-name}]
-            header]
+           [ui/ModalHeader [ui/Icon {:name icon-name}] header "\u2001\u00a0"]
            (when content
-             [ui/ModalContent
-              {:scrolling true}
+             [ui/ModalContent {:scrolling true}
               [:pre content]])])))))
 
 
-(defn message-item-card
+(defn message-item
   [locale index {:keys [type header content timestamp]}]
   (let [visible? (reagent/atom false)]
     (fn [locale index {:keys [type header content timestamp]}]
-      (let [header-class (str "webui-" (name type))]
-        [ui/Card {:fluid true}
-         [ui/Label {:as       :a
-                    :corner   "right"
-                    :size     "mini"
-                    :icon     (if @visible? "chevron down" "chevron up")
-                    :on-click #(reset! visible? (not @visible?))}]
-         [ui/Label {:as       :a
-                    :corner   "left"
-                    :size     "mini"
-                    :icon     "close"
-                    :on-click #(dispatch [::message-events/remove index])}]
-         [ui/CardContent {:class-name header-class}
-          [ui/CardHeader
-           [ui/Icon {:name (type->icon-name type)}]
+      (let [icon-name (type->icon-name type)]
+        [ui/ListItem {:on-click #(reset! visible? (not @visible?))}
+         [ui/ListIcon {:name icon-name, :size "large"}]
+         [ui/ListContent
+          [ui/Button {:floated  "right"
+                      :size     "tiny"
+                      :icon     "close"
+                      :on-click (fn [e]
+                                  (when e
+                                    (.stopPropagation e true))
+                                  (dispatch [::message-events/remove index]))}]
+          [ui/ListHeader header]
+          [ui/ListDescription (time/ago timestamp locale)]]
+
+         [ui/Modal
```

Comment:
click on modal cross icon to close doesn't work, escape key work

Reply:
Will check why this is happening.